src/components/header/search/Search.jsx

```diff
@@ -1,15 +1,20 @@
+import { useLocation } from "react-router-dom";
+
 export function Search({ searchMob }) {
+  const location = useLocation();
   return (
     <div className={`sm-nav-search${searchMob ? "-mobile" : ""}`}>
-      <div className="sm-input-container input-search">
-        <input
-          type="text"
-          className="input-basic input-basic-icon"
-          placeholder="Search..."
-          name="search"
-        />
-        <i className="icon-basic fas fa-search"></i>
-      </div>
+      {location.pathname === "/videos" && (
+        <div className="sm-input-container input-search">
```

Comment:
Suggestion: extract out boolean logic for improved readability and ease of maintenance.

Example: 
`const isVideoPage = location.pathname === "/videos"`